src/main/java/org/terasology/las/UI/ScoreHud.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.las.UI;
+
+import org.terasology.engine.Time;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.ligthandshadow.componentsystem.components.ScoreComponent;
+import org.terasology.logic.health.HealthComponent;
+import org.terasology.logic.location.LocationComponent;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
+import org.terasology.rendering.nui.layers.hud.CoreHudWidget;
+import org.terasology.rendering.nui.widgets.UIText;
+
+public class ScoreHud extends CoreHudWidget {
+    @In
+    private EntityManager entityManager;
+
+    private UIText scoreArea;
+    private EntityRef scoreEntity;
+    private ScoreComponent score;
+
+    @Override
+    public void initialise() {
+        for (EntityRef entity : entityManager.getEntitiesWith(ScoreComponent.class)) {
+            scoreEntity = entity;
```

Comment:
Are you expecting one or more entities with `ScoreComponent.class`? If it is more than one, why store one of them (the last one) as `private EntityRef scoreEntity;`
Currently, the variable is only accessed in the local scope of `initialise`.